package.json

```diff
@@ -26,6 +26,7 @@
     "coffee-script": "1.10.0",
     "coffeelint": "1.15.7",
     "gently": "0.10.0",
+    "localtunnel": "^1.8.1",
```

Comment:
We're a big fan of pinning exact versions. If you disagree let me know, and we could exchange ideas about this in slack